src/lando/dockerflow/tests/conftest.py

```diff
@@ -0,0 +1 @@
+from lando.tests.conftest import lando_version  # noqa: F401

```

Comment:
nit: another (more conventional) way to prevent the lint error is to add the import to `__all__`.
```suggestion
from lando.tests.conftest import lando_version

__all__ = ["lando_version"]
```